app/models/concerns/client_search.rb

```diff
@@ -51,6 +51,14 @@ def self.text_searcher(text, sorted:, resolve_for_join_query: false)
           )
           where = conditions.reduce(:or)
         end
+      elsif /household:[[[:alnum:]]-]+/.match(text)
+        household_id = text.gsub('household:', '')
```

Comment:
Nice! Some thoughts, feel free to ignore
* assuming we are storing normalized uuids, adjust the ID string so it will match by upcasing and removing hyphens. This lets someone search with a formatted uuid
* limit to some max len (uuids should be 32)
* match start/end of line to ensure there isn't other cruft in term that we don't support
```suggestion
      elsif text =~ /\Ahousehold:([-[:alnum:]]+)\z/
        household_id = $1.gsub('-', '').upcase.first(32)
```

Reply:
They are not necessarily UUIDs. Data imported from other system could have numeric Household IDs for example. I agree with removing dashes since we do that in the PersonalID search too, I'll add that change. & the start/end match. Thanks!